app/Models/Game.php

```diff
@@ -114,12 +115,15 @@ public function get_users()
 
     public function get_users_filtered_on_last_verified()
     {
-        if($this->last_interval_at == null) {
+        if ($this->last_interval_at == null) {
             return $this->get_users_with_role()->all();
         }
 
+        // Because of the delay and the interval, sometimes user last_interval_at will be updated to lately
+        $dateTime = Carbon::parse($this->last_interval_at)->addSeconds(-30)->toDateTimeString();
```

Comment:
Ik ga nu 30 secondes terug in de tijd, want sommige apparaten zijn wat later met hun `last_verified_at` te bijwerken, waardoor het lijkt alsof ze telkens een paar secondes disconencted zijn.